x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/esnative/tool/SetupPasswordTool.java

```diff
@@ -134,6 +138,12 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
             checkClusterHealth(terminal);
 
             if (shouldPrompt) {
+                terminal.println("******************************************************************************");
+                terminal.println("Note: The 'elasticsearch-setup-passwords' tool has been deprecated");
+                terminal.println("      and might be removed in a future release.");
+                terminal.println("      Use the 'elasticsearch-reset-elastic-password' tool instead. ");
```

Comment:
See my comment above - not every use of setup-passwords can switch to "elasticsearch-reset-elastic-password" and I worry about confusion.